### PR TITLE
Parse int

### PR DESCRIPTION
**Pull request by barisusakli**
for tests

### CODE REVIEW

File: src/controllers/accounts/helpers.js [hunk 1]
```diff
@@ -159,15 +159,15 @@ helpers.getUserDataByUserSlug = function (userslug, callerUID, callback) {
...
+			userData.canChangePassword = isAdmin || (isSelf && !meta.config['password:disableEdit']);
 			userData.isSelf = isSelf;
 			userData.isFollowing = results.isFollowing;
 			userData.showHidden = isSelf || isAdmin || (isGlobalModerator && !results.isTargetAdmin);
 			userData.groups = Array.isArray(results.groups) && results.groups.length ? results.groups[0] : [];
-			userData.disableSignatures = meta.config.disableSignatures !== undefined && parseInt(meta.config.disableSignatures, 10) === 1;
-			userData['reputation:disabled'] = parseInt(meta.config['reputation:disabled'], 10) === 1;
-			userData['downvote:disabled'] = parseInt(meta.config['downvote:disabled'], 10) === 1;
-			userData['email:confirmed'] = !!parseInt(userData['email:confirmed'], 10);
+			userData.disableSignatures = meta.config.disableSignatures === 1;
```
Comment (pitaj) [inline src/controllers/accounts/helpers.js]: Most of the `=== 1` settings should probably be booleans instead

File: src/meta/configs.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ function deserialize(config) {
 	Object.keys(config).forEach(function (key) {
 		if (utils.isNumber(config[key])) {
```
Comment (pitaj) [inline src/meta/configs.js]: This breaks boolean conditions. Better to just use `try ... catch`

File: src/meta/configs.js [hunk 3]
```diff
@@ -180,7 +180,7 @@ function updateConfig(config) {
 }
 
 function updateLocalConfig(config) {
-	Object.assign(Meta.config, config);
```
Comment (pitaj) [inline src/meta/configs.js]: Should probably just send the already-deserialized config between processes instead.

Comment (barisusakli) [reply src/meta/configs.js]: I changed this line to `Object.assign(Meta.config, deserialize(config));` so when configs are saved in ACP they dont go back to strings.

**Pull request merged**